galvan-support/src/main/java/org/terracotta/testing/rules/BasicExternalCluster.java

```diff
@@ -180,30 +182,29 @@ private void internalStart() throws Throwable {
 
     Path tcConfig = createTcConfig(serverNames, serverPorts, serverGroupPorts, stripeInstallationDir);
 
-    StripeConfiguration stripeConfig = new StripeConfiguration(kitDir.toAbsolutePath(), stripeInstallationDir,
-        serverDebugPorts, serverPorts, serverGroupPorts, serverNames, stripeName, serverJars, DEFAULT_SERVER_HEAP_MB,
+    Path kitLocation = installKit(stripeVerboseManager, kitDir, serverJars, stripeInstallationDir);
+
+    StripeConfiguration stripeConfig = new StripeConfiguration(serverDebugPorts, serverPorts, serverGroupPorts, serverNames, stripeName, DEFAULT_SERVER_HEAP_MB,
         logConfigExt, systemProperties);
     StripeInstaller stripeInstaller = new StripeInstaller(interlock, stateManager, stripeVerboseManager, stripeConfig);
     // Configure and install each server in the stripe.
     for (int i = 0; i < stripeConfig.getServerNames().size(); ++i) {
```

Comment:
Unrelated to this PR, but `stripeConfig.getServerNames().size()` can be simplified to just say `stripeSize`